src/Helper/GitHelper.php

```diff
@@ -727,11 +727,7 @@ public function getGitDir()
             return $this->topDir;
         }
 
-        try {
-            $this->topDir = $this->processHelper->runCommand(['git', 'rev-parse', '--show-toplevel']);
-        } catch (\RuntimeException $e) {
-            $this->topDir = '';
-        }
+        $this->topDir = $this->processHelper->runCommand(['git', 'rev-parse', '--show-toplevel']);
```

Comment:
All usage of this method throws an exception, but the message should be correct.

Sometimes the User exception hides the details of the failure, like an config file with invalid content.